fmralign/alignment_methods.py

```diff
@@ -6,8 +6,9 @@
 from scipy.spatial.distance import cdist
 from scipy import linalg
 from scipy.sparse import diags
+import sklearn
 from sklearn.base import BaseEstimator, TransformerMixin
-from sklearn.utils.linear_assignment_ import linear_assignment
+from scipy.optimize import linear_sum_assignment
```

Comment:
Does it do exactly the same thing ?

Reply:
yes sklearn people are using it as well in place of their old method and deprecating it in next versions.

Reply:
https://github.com/scikit-learn/scikit-learn/issues/13464

Reply:
It returns a tuple of arrays rather than one array but I [added another line](https://github.com/Parietal-INRIA/fmralign/pull/43/files#diff-130259fe9c0b0a8dd336f7690331449aR87) to update the handling appropriately !